packages/babel-plugin-minify-builtins/src/index.js

```diff
@@ -123,6 +124,26 @@ module.exports = function({ types: t }) {
 
 };
 
+// decides the best path to insert the newly created identifier
+function getPath(program, firstPath, lastPath) {
+  let resultPath = program;
+  // check if occurence of member exp ex - Max.max() is before the
+  // block statement container
+  const { start: nodeStart } = firstPath.parent;
+  const { end: nodeEnd } = lastPath.parent;
+
+  program.traverse({
```

Comment:
the same can be achieved by `getFunctionParent` recursively till `program`. `program.traverse` inside a loop will be costly.